components/occurrence/List.vue

```diff
@@ -90,8 +91,9 @@ const occurrences = computed(() => {
     return filters.filterOccurrences(data.value.occurrences)
 
   const unavailable = data.value.occurrences.filter(o => o.notAvailableAfter !== null)
-  const { visible, hidden } = filters.filterOccurrences(data.value.occurrences.filter(o => o.notAvailableAfter === null))
-  return { visible, hidden: [...hidden, ...unavailable] }
+  const {visible, hidden} = filters.filterOccurrences(data.value.occurrences.filter(o => o.notAvailableAfter === null))
+
+  return {visible: sorting.sortOccurrences(visible), hidden: sorting.sortOccurrences([...hidden, ...unavailable])}
```

Comment:
I don't really like how complex this method is getting but I guess there's no way around it...